### DISCUSSION TITLE
part-ii/2018-100/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#300] [Question comments]
Part II 2018 Number Theory: Paper 2, Section I, G 

https://questions.tripos.org/part-ii/2018-100/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

Happily (for me typing this up), most of this is bookwork. The last bit is _almost_ bookwork, as there's a few examples given in lectures and example sheets, and it's just applying all the results we know about manipulating Legendre symbols. I got $-1$, and a Legendre symbol calculator online backs me up here.

</details>
